modules/ovn-kubernetes-logical-architecture-con.adoc

```diff
@@ -0,0 +1,68 @@
++
+[source,terminal]
+----
+*     "GR_ip-10-0-206-73.us-east-2.compute.internal" (b6626ff9-8d06-4dc5-9665-ed34815f732d) [ingress]
+  *     "ovn_cluster_router" (d5fc1961-658e-4b2d-acf2-70c3b4584b8c) [ingress]
+  *     "GR_ip-10-0-140-161.us-east-2.compute.internal" (e44aa51c-21ff-4cda-b748-89a6f18a6da6) [ingress]
+----
+Join local switch:: Join local switches are used to connect the distributed router and gateway routers. It reduces the number of IP addresses needed on the distributed router. For example in the following `ovnkube-trace` snippet the join logical switch is identified as `join`.
++
+[source,terminal]
+----
+* Logical datapaths:
+  *     "ip-10-0-149-128.us-east-2.compute.internal" (0255bd08-b156-4b50-a078-a5d6b06208b9) [ingress]
+  *     "ip-10-0-140-161.us-east-2.compute.internal" (12d1e5b1-5346-43b5-b723-6fa4c9b07445) [ingress]
+  *     "ip-10-0-206-73.us-east-2.compute.internal" (398ca3e4-e70b-4632-af4e-64f88e17c133) [ingress]
+  *     "join" (5db354c9-37b6-4139-beab-25fa009bcb16) [ingress]
+  *     "ext_ip-10-0-156-14.us-east-2.compute.internal" (70b4031c-bc50-47e6-bcc5-94aae13c4ca5) [ingress]
+  *     "ext_ip-10-0-206-73.us-east-2.compute.internal" (98366c43-7d8d-4b6d-8d83-0a64a68175a2) [ingress]
+----
+
+Logical switches with patch ports:: Logical switches with patch ports are used to virtualize the network stack. They connect remote logical ports through tunnels.
+
+Logical switches with localnet ports:: Logical switches with localnet ports are used to connect OVN to the physical network. They connect remote logical ports by bridging the packets to directly connected physical L2 segments using localnet ports. For example in the following `ovnkube-trace` snippet the join logical switches with localnet ports are prefixed with `ext_`.
++
+[source,terminal]
+----
+* Logical datapaths:
+  *     "ip-10-0-149-128.us-east-2.compute.internal" (0255bd08-b156-4b50-a078-a5d6b06208b9) [ingress]
+  *     "ip-10-0-140-161.us-east-2.compute.internal" (12d1e5b1-5346-43b5-b723-6fa4c9b07445) [ingress]
+  *     "ip-10-0-206-73.us-east-2.compute.internal" (398ca3e4-e70b-4632-af4e-64f88e17c133) [ingress]
+  *     "join" (5db354c9-37b6-4139-beab-25fa009bcb16) [ingress]
+  *     "ext_ip-10-0-156-14.us-east-2.compute.internal" (70b4031c-bc50-47e6-bcc5-94aae13c4ca5) [ingress]
+  *     "ext_ip-10-0-206-73.us-east-2.compute.internal" (98366c43-7d8d-4b6d-8d83-0a64a68175a2) [ingress]
+  *     "ip-10-0-156-14.us-east-2.compute.internal" (b861d78c-e7b9-4122-8207-6eba3f259057) [ingress]
+  *     "ip-10-0-147-232.us-east-2.compute.internal" (c42f5b0c-84b2-4f54-8d55-aef09a124e66) [ingress]
+  *     "ext_ip-10-0-149-128.us-east-2.compute.internal" (d5481d3e-fac2-47b7-a218-71497dbc6e33) [ingress]
+----
+
+Patch ports:: Patch ports represent connectivity between logical switches and logical routers and between peer logical routers. A single connection has a pair of patch ports at each such point of connectivity, one on each side.
```

Comment:
To find the 'patch' ports:
./network-tools ovn-db-run-command ovn-sbctl find Port_Binding type=patch

Reply:
@CFields651 I do tend to agree with you, but I believe we discourage the installation of third party tool in our documentation. I know they are support concerns around using jq so I expect there may be issues with network-tools though maybe not. I will check with OCP doc team